src/main/java/study/domain/Operator.java

```diff
@@ -0,0 +1,51 @@
+package study.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (first, second) -> first + second),
+    MINUS("-", (first, second) -> first - second),
+    MULTIPLY("*", (first, second) -> first * second),
+    DIVIDE("/", (first, second) -> {
+        if (second == 0) throw new IllegalArgumentException("0으로 나눌 수 없습니다.");
+        return first / second;
+    });
+
+    private String symbol;
+    private BiFunction<Integer, Integer, Integer> operation;
+
+    Operator(final String symbol, final BiFunction<Integer, Integer, Integer> operation) {
+        this.symbol = symbol;
+        this.operation = operation;
+    }
+
+    /**
+     * 캐싱해서 순회 피하기
+     */
+    private static final Map<String, Operator> SYMBOL_MAP = new HashMap<>();
+
+    static {
+        for (Operator operator : values()) {
+            SYMBOL_MAP.put(operator.symbol, operator);
+        }
+    }
+
+    public static Operator getSymbol(final String param) {
+        if (SYMBOL_MAP.containsKey(param) == false) {
```

Comment:
```suggestion
        if (!SYMBOL_MAP.containsKey(param)) {
```

== 비교 없이 작성이 가능하겠네요 😄

Reply:
이건 개인 취향 입니다 :) 
== false 
!true 

저는 == false 도 가독성상 좋다고 생각해요 :)